src/sidebar/store/test/index-test.js

```diff
@@ -61,6 +61,63 @@ describe('store', function() {
     });
   });
 
+  describe('clearSelection', () => {
+    // Test clearSelection here over the entire store as it triggers the
+    // CLEAR_SELECTION action in multiple store modules.
```

Comment:
Hmm ... I've mixed thoughts about this. On the one hand, it is useful that all the tests for this function are in one place. On the other, someone looking at just the tests for a particular module might think that this action is untested.

Reply:
Yeah I did too but I felt like the integrationy store tests should go in the index and while some of these tests could have gone in the corresponding individual modules I thought it was simpler to keep them all in one place. We could put a comment into the individual modules saying look here for these tests or something.